packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -66,6 +67,7 @@ export class TSClient implements Generatable {
       runtimeName,
       datasources,
       dataProxy,
+      deno,
```

Comment:
Question: how does `--deno` and `--data-proxy` interact with each other? Does `--deno` imply `--data-proxy`? Or are they incompatible? If latter, will we have an error if they are both specified?

Reply:
`--deno` does not exist, only `deno` in the list of preview features in the schema.

Reply:
Got it, thanks

Reply:
The Deno CLI is detected, on top of that. So 3 requirements at the moment